config/mongoSchema.js

```diff
@@ -0,0 +1,65 @@
+let mongoose = require('mongoose');
+mongoose.Promise = global.Promise;
+// (node:26944) DeprecationWarning: current URL string parser is deprecated, and will be removed in a future version. To use the new parser, pass option { useNewUrlParser: true } to MongoClient.connect.
+mongoose.connect("mongodb://127.0.0.1/hacknews", {config: {autoIndex: false}}, function(err){
+  if(err){ // todo: crash/panic.
+    console.log('err connecting mongoDB: ', err)
+  } else {
+    console.log('Connected to mongodb!')
+  }
+});
+
+let Schema = mongoose.Schema,
+    ObjectId = Schema.ObjectId;
+
+let UserSchema = new Schema({
+  versionKey: false,
+  /*
+  email: {
+    type: String,
+    unique: true,
+    required: true,
+    trim: true
+  },
+  */
```

Comment:
could you let me know why you comment out those codes?

Reply:
No specific reasons, most website allow users to use email, cellphone number, or username as login username. Maybe I'd add this feature later.